tests_metricflow/snapshots/test_derived_metric_rendering.py/SqlQueryPlan/DuckDB/test_derived_metric_that_defines_the_same_alias_in_different_components__plan0_optimized.sql

```diff
@@ -0,0 +1,59 @@
+-- Combine Aggregated Outputs
+-- Order By [] Limit 1
+SELECT
+  COALESCE(subq_18.booking__is_instant, subq_24.booking__is_instant) AS booking__is_instant
+  , MAX(subq_18.derived_sharedalias_1a) AS derived_sharedalias_1a
+  , MAX(subq_24.derived_sharedalias_2) AS derived_sharedalias_2
+FROM (
+  -- Compute Metrics via Expressions
+  SELECT
+    booking__is_instant
+    , shared_alias - 10 AS derived_sharedalias_1a
```

Comment:
Do these SQL queries run as expected even though we use the `bookings_source_src_28000` alias twice ?

Reply:
I think it's fine since the alias is defined separately in each subquery so the scope is different

Reply:
Yeah ok I think it should be fine!